### DISCUSSION TITLE
Welcome to Miner Legacy Fork Fix (for NiceHash) Discussions!

### DISCUSSION THREAD

**Discussion by angelbbs** [angelbbs/NiceHashMinerLegacy#100] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phanity2** [comment]
When is the new version launching?

**Comment by angelbbs** [comment]
as soon as possible

**Reply by Andymc1280** [reply]
> 
> 
> as soon as possible

Plz Plz Plz keep the old logo!  😁
